clk-gateway/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "clk-gateway",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "@types/node": "^22.9.4",
+    "dotenv": "^16.4.5",
+    "ethers": "^6.13.4",
+    "express": "^4.21.1",
+    "firebase": "^11.0.2",
+    "firebase-admin": "^13.0.1",
+    "typescript": "^5.7.2",
+    "zksync-ethers": "^6.15.1"
+  },
+  "devDependencies": {
+    "@types/express": "^5.0.0",
+    "@types/jest": "^29.5.14",
+    "jest": "^29.7.0",
+    "nodemon": "^3.1.7",
+    "prettier": "^3.3.3",
+    "ts-jest": "^29.2.5",
+    "ts-node": "^10.9.2"
+  },
+  "scripts": {
+    "fmt": "prettier --write src/**/*.ts",
+    "test": "jest",
+    "start": "node dist/index.js",
+    "dev": "nodemon src/index.ts",
+    "build": "tsc",
+    "serve": "yarn build && yarn start",
+    "createToken": "node dist/createToken.js"
```

Comment:
We should use `tsnode` instead of the `node dist/...` pattern

Reply:
done